spring-pulsar/src/main/java/org/springframework/pulsar/listener/reactive/ReactivePulsarContainerProperties.java

```diff
@@ -49,6 +49,8 @@
 
 	private int maxInFlight = 0;
 
+	private boolean useKeyOrderedProcessing = false;
```

Comment:
This reminds me that there is probably nothing stopping us from turning this into a simple `record`.